django/core/management/commands/compilemessages.py

```diff
@@ -154,7 +154,9 @@ def compile_messages(self, locations):
                     self.has_errors = True
                     return
 
-                args = [self.program, *self.program_options, '-o', mo_path, po_path]
+                # TODO: Remove str() when dropping support for PY37.
```

Comment:
Could we use `PY37` here instead of `TODO`? (Thinking that when we come to remove the PY37 constant it'll at least show up in the search...) 

(Not a biggie.)